templates/ts-apollo-fullstack/client/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "echo disabled",
+    "build": "react-scripts build",
```

Comment:
This should be typescript build

Reply:
Are you sure about that? Because I tried that first and it did nothing. Would `tsc` compile the HTML and CSS too?

Reply:
Hmm I wonder if it is because these are tsx files. I will check.

Reply:
Actually, we might remove build completely from client, but put build into server etc. Trick is to remove echo cmd that look quite unprofessional when app is bootstrapped